srcpkgs/matterhorn/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'matterhorn'
+pkgname=matterhorn
+version=50200.7.0
+revision=1
+makedepends="cabal-install ghc zlib-devel ncurses-devel"
+depends="gmp zlib"
+short_desc="A terminal-based chat client for MatterMost"
+maintainer="Stacy Harper <contact@stacyharper.net>"
+license="BSD-3-Clause"
+homepage="https://github.com/matterhorn-chat/matterhorn"
+changelog="https://raw.githubusercontent.com/matterhorn-chat/matterhorn/master/CHANGELOG.md"
+distfiles="https://github.com/matterhorn-chat/matterhorn/archive/${version}.tar.gz"
+checksum=24ee5ac95c37e30e209ed6d39f4e9986c2f37365382e4f56a8e1988516f26ff3
+nopie_files="/usr/bin/matterhorn"
+
+do_build() {
+	rm -f cabal.project
+	cabal new-update
+	cabal new-build --jobs="${XBPS_MAKEJOBS}" --disable-executable-dynamic
+}
+
+do_install() {
+	vbin dist-newstyle/build/x86_64-linux/ghc-8.6.5/matterhorn-50200.7.0/x/matterhorn/build/matterhorn/matterhorn
```

Comment:
This won't work, xbps supports more architectures than just x86_64. I'm sure cabal has better support for installing binaries than that.

Reply:
I'm trying to use cabal with some sort of distdir or bindir. I leave it this way for tonight. I'll try again tomorrow, reading seriously documentations.